### ISSUE TITLE
Log not showing problematic file name for invalid JPEG format 

### ISSUE THREAD

**Issue by guysoft** [photoprism/photoprism#210]
Log is reporting an issue with a file, but I can't track down which:
```

2020-01-21 01:24:36 ERRO thumbs: invalid JPEG format: missing 0xff00 sequence

2020-01-21 01:24:36 ERRO thumbs: can't open original "invalid JPEG format: missing 0xff00 sequence"

2020-01-21 01:17:43 INFO indexing photos in "Originals"

2020-01-21 01:17:29 ERRO thumbs: canceled

2020-01-21 01:05:17 ERRO thumbs: invalid JPEG format: missing 0xff00 sequence

2020-01-21 01:05:17 ERRO thumbs: can't open original "invalid JPEG format: missing 0xff00 sequence"

2020-01-21 12:58:14 INFO indexing photos in "Originals"

2020-01-21 12:14:51 ERRO thumbs: invalid JPEG format: missing 0xff00 sequence

2020-01-21 12:14:51 ERRO thumbs: can't open original "invalid JPEG format: missing 0xff00 sequence"
```

**Comment by lastzero** [comment]
As a quick fix, you can run the app in debug mode which will show more information in the logs

**Comment by lastzero** [comment]
I'll merge this to master when CI passes so that you can use our image and don't need to build from develop.

**Comment by lastzero** [comment]
Build is running (also for ARM64): https://travis-ci.org/photoprism/photoprism/builds/639974649

**Comment by guysoft** [comment]
@lastzero cool! What would be the docker image to pull once this is finished to test?

**Comment by lastzero** [comment]
Always `latest` or the one with the current date as tag: `20200121`

For arm64, simply add arm64 like `photoprism/photoprism-arm64:latest`

**Comment by lastzero** [comment]
@guysoft Does this work for you now?

**Comment by guysoft** [comment]
Importing ~20k (and growing) images on a RaspberryPi 4, its been still going. Will do once its finished and will update.

**Comment by lastzero** [comment]
Did you find the invalid JPEGs? Could you open them in Photoshop or other tools?

Would be great if you can send samples for testing.

**Comment by lastzero** [comment]
New build for arm64 is running...

The `index` command does not re-index all files by default anymore (added `--all` flag). Command help was also improved.

**Comment by guysoft** [comment]
> The `index` command does not re-index all files by default anymore (added `--all` flag). Command help was also improved

What does this mean? What should I do when I index?

**Comment by lastzero** [comment]
@guysoft You have to do nothing, just FYI. See #187 for details.

**Comment by guysoft** [comment]
Ok, it reached 30K images, so I became impatient and killed the container, pulled yours, restarted it and now indexing to catch this bug.

**Comment by lastzero** [comment]
Closing this in favor of #217 as it's similar and the log messages were obviously improved as you can see from the commit history.
